lib/internal/modules/esm/get_format.js

```diff
@@ -15,7 +15,7 @@ const {
   mimeToFormat,
 } = require('internal/modules/esm/formats');
 
-const detectModule = getOptionValue('--experimental-detect-module');
+const detectModule = !getOptionValue('--no-experimental-detect-module');
```

Comment:
maybe we should add `hasOptionValue()` function that returns on boolean?

Reply:
What does `getOptionValue('--experimental-detect-module')` actually return?

Reply:
boolean

Reply:
I mean,the flag is now enabled by default. I won't block but I don't find `if(!disabled)` easier to understand than `if(enabled)`

Reply:
you are right I dropped the commit